site/content/entry/yuzu-progress-report-jan-2023/index.md

```diff
@@ -0,0 +1,491 @@
+  >}}
+
+Those settings got lost every time a game was booted.
+Well, the issue was that those settings were being saved on the per-game configuration files for no reason.
+There’s no reason to include a fundamentally global setting in the per-game settings, so {{< gh-hovercard "9521" "moving them away" >}} solved the issue.
+
+byte[] solved an issue making accessing fullscreen crash, sorry, let’s say it like the bug report, making yuzu CRASH!!
+{{< gh-hovercard "9601" "Adjusting the behaviour of yuzu’s bootmanager" >}} was enough to stop the CRASH!!
+
+Newcomer [SaiKai](https://github.com/SaiKai) implements one of those Quality of Life changes that you can’t live without and you can’t believe it wasn’t a thing before.
+Now pressing and holding the hotkey assigned to lowering or raising {{< gh-hovercard "9637" "volume will auto-repeat," >}} making it much easier to control.
+Thanks!
+
+Per-game configuration returns to mess with us, now with the language selection.
+The language combobox index gets altered in per-game configuration, so setting the region to Korea or Taiwan would result in a crash.
+german77 {{< gh-hovercard "9660" "fixes this discrepancy" >}} solving the issue.
+
+## Hardware section
+
+Let’s revive this section for a bit, there’s some information I’d like to discuss.
+
+#### NVIDIA, forgetting about their old cards.
+
+We previously mentioned that the 527.XX release of drivers solved issues we previously had with Pascal and newer cards.
+While this was true for most issues, Pascal is still hitting some of the old problems, like crashes with ASTC texture decoding enabled.
+We recommend Pascal users to either disable ASTC texture decoding, or revert back to 512.95 drivers.
+We hope NVIDIA backports the fixes Turing and newer cards got to the old guard too.
+
+#### AMD, custom drivers are not good
+
+The recently released RX 7000 series, or RDNA3, are not particularly good with emulators right now, with yuzu not being an exception.
+Broken rendering and crashes were reported by early adopters.
+
+Sadly, we can’t focus on these products while they run a custom driver branch. 
+Until AMD merges the RDNA3 driver with the regular Polaris to RDNA2 one we’re all used to, it’s impossible to focus efforts on trying to solve any particular issues of this new release.
+Sorry early adopters, you will have to wait.
+
+Having access to hardware that doesn’t cost NVIDIA-stupid levels would also help. 
+AMD, please don’t take a year to release the mid and low end.
+
+Now, regarding non-official custom drivers, we got reports that Amermine Zone drivers break rendering in many games. 
+Regular drivers are perfectly fine, so just stick to official AMD releases.
+
+#### Intel, artificial limitations capping good hardware
+
+Your writer recently had to replace an old i3 5005U laptop because its WiFi died, HP only allows a very limited selection of cards as replacements, and such parts are no longer in production.
```

Comment:
```suggestion
Your writer recently had to replace an old i3-5005U laptop because its WiFi died, but HP only allows a very limited selection of cards as replacements, and said parts are no longer in production.
```